packages/terra-card/src/Card.jsx

```diff
@@ -12,20 +12,31 @@ const propTypes = {
    * Child Nodes
    */
   children: PropTypes.node,
+  /**
+   * An option used in order to change to a different style of card for different use cases
+   */
+  variant: PropTypes.oneOf(['default', 'raised']),
```

Comment:
I couldn't find a pattern on what the prefered way to do this is. I see
* just an array inline like this
* a const array up above and then used here
* an object where the keys and values are all caps and then doing object.values here
* adding `opts` to the component, exporting, and using those.

Let me know if there is a preference!

Reply:
I'm currently working on #1044. I volunteered to go through all the components and do the uplift to allow for exported enumeration of variants, I'd prefer it be done following [this](https://github.com/cerner/terra-core/commit/8bae7969950a3868334cf871b72da372dea373ee) pattern, as it's how I plan on implementing all component enumerations.

Reply:
I like the use of enums, but I think there needs to be a conversation on the consumption of them. Because it's easier and more terse to do something like `<Card variant="raised" ... />` than `<Card variant={Card.Opts.Variants.RAISED} ... />` as a consumer. I get why the precedence is to put it behind another `Opts` object, but it just adds to verbosity and feels like writing Java all over again. 

`Card.Variants.RAISED` >>> `Card.Opts.Variants.RAISED`, in my opinion

You could also name export `Variants` or `CardVariants` from here, but `CardVariants` is only one key stroke less than `Card.Variants` and `Variants` will need to be renamed by the consumer if they use multiple terra components in the same file (all too common), e.g. `import { Variants as CardVariants } from 'terra-card';

The above can also be done with destructuring from the Card like `const { Variants } = Card` or renaming it like `const { Variants: CardVariants } = Card;`

I don't putting the burden of renaming an enum on the consumer and I think if getting to the enum takes too long a consumer is just going to do `<Card variant="raised" ... />`

Reply:
Updated